docs/charts.md

```diff
@@ -302,6 +302,108 @@ helm install --set tags.front-end=true --set subchart2.enabled=false
   * The `tags:` key in values must be a top level key. Globals and nested `tags:` tables
     are not currently supported.
 
+#### Importing Child Values via requirements.yaml
+
+In some cases it is desirable to allow a child chart's values to propagate to the parent chart and be 
+shared as common defaults. An additional benefit of using the `exports` format is that it will enable future 
+tooling to introspect user settable values.
+
+The keys containing the values to be imported can be specified in the parent chart's requirements.yaml using 
```

Comment:
- [ ] Elaborate on what "requirements.yaml" means.  
  Set it in italic and use the word "file."  
  > the parent chart's _requirements.yaml_ file